P0009/P0009.bs

```diff
@@ -1805,28 +1796,16 @@ namespace std {
 namespace experimental {
 namespace fundamentals_v3 {
 
-template<class T>
 struct accessor_basic {
```

Comment:
The gain is to be able to define and compose access properties independently of the type to which they are applied; e.g., atomic, streaming, etc.  Cumbersome to compose access properties if the ElementType must be carried along during composition, as apposed to being applied after the fact.  Consider a "rebind" or "apply to" interface:
  `auto ax = apply_property( x , access_atomic );`
where
  `inline constexpr const access_atomic_t access_atomic;`

Reply:
Actually, the code you gave originally:
```
auto ax = apply_property( x , access_atomic );
```
can just work as is, assuming an implementation of `apply_property` that deduces the type of `x` and acts on some sentinel `access_atomic`, especially if we add an optional `rebind` member template, like on allocator. I wouldn't be opposed to that, because that's something I think we could reasonably allow at the concept level without losing the ability to allow accessors to restrict the types they act on and partially specialize.